sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -24,10 +25,32 @@ export const ContentPlacementContainerStyles = css`
         grid-template-columns: repeat(auto-fit, minmax(225px, 1fr));
     }
 
+    /* this creates the numbering for feature*/
```

Comment:
Could this be implemented in a more reusable fashion? For example, a named slot rather than limiting it to a section?  Then it would just be a matter of including the counter (or id or ordinal), slot in any given templates that need a visible number.

Reply:
discussing potential solutions for this on discord

Reply:
I think making the feature card numbering in this situation more reusable/generic is counter to the part where we're already calling it a `feature-card`. If there were situations where feature cards didn't have numbers, I'd say abstracting it out would make sense.